sdv/datasets/demo.py

```diff
@@ -125,10 +132,43 @@ def download_demo(modality, dataset_name, output_folder_name=None):
             * If there is already a folder named ``output_folder_name``.
             * If ``modality`` is not ``'single_table'``, ``'multi_table'`` or ``'sequential'``.
     """
-    _validate_args_download_demo(modality, output_folder_name)
+    _validate_modalities(modality)
+    _validate_output_folder(output_folder_name)
     bytes_io = _download(modality, dataset_name)
     in_memory_directory = _extract_data(bytes_io, output_folder_name)
     data = _get_data(modality, output_folder_name, in_memory_directory)
     metadata = _get_metadata(modality, output_folder_name, in_memory_directory)
 
     return data, metadata
+
+
+def get_available_demos(modality):
+    """Get demo datasets available for a ``modality``.
+
+    Args:
+        modality (str):
+            The modality of the dataset: ``'single_table'``, ``'multi_table'``, ``'sequential'``.
+
+    Returns:
+        pandas.DataFrame:
+            A table with three columns:
+                ``dataset_name``: The name of the dataset.
+                ``size_MB``: The unzipped folder size in MB.
+                ``num_tables``: The number of tables in the dataset.
+
+    Raises:
+        Error:
+            * If ``modality`` is not ``'single_table'``, ``'multi_table'`` or ``'sequential'``.
+    """
+    _validate_modalities(modality)
+    client = boto3.client('s3', config=Config(signature_version=UNSIGNED))
```

Comment:
As we discussed in my last meeting last year, I updated the previous logic (which used `requests` and `ET`) to use boto.